src/main/java/org/mastodon/mamut/feature/branch/BranchDisplacementDurationFeature.java

```diff
@@ -47,6 +40,117 @@
 import org.mastodon.properties.DoublePropertyMap;
 import org.scijava.plugin.Plugin;
 
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
```

Comment:
Please avoid resorting the import statements.

Reply:
Changed it.